<launch>
  <arg name="input/trajectory" default="/planning/scenario_planning/trajectory" />
  <arg name="input/reference_trajectory" default="/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/trajectory" />
  <arg name="input/objects" default="/perception/object_recognition/objects" />

  <node name="planning_evaluator" exec="planning_evaluator" pkg="planning_evaluator" output="screen">
    <param from="$(find-pkg-share planning_evaluator)/param/planning_evaluator.defaults.yaml"/>
    <remap from="~/input/trajectory" to="$(var input/trajectory)" />
    <remap from="~/input/reference_trajectory" to="$(var input/reference_trajectory)" />
    <remap from="~/input/objects" to="$(var input/objects)" />
  </node>
</launch>
